src/smoosh/src/smoosh_channel.erl

```diff
@@ -105,8 +105,9 @@ init(Name) ->
     erlang:send_after(60 * 1000, self(), check_window),
     process_flag(trap_exit, true),
     Waiting = smoosh_priority_queue:new(Name),
+    Persist = config:get_boolean("smoosh", "persist", false),
     State =
-        case smoosh_utils:is_view_channel(Name) of
+        case smoosh_utils:is_view_channel(Name) orelse Persist =:= "false" of
```

Comment:
`Persist` value would be a boolean so you'd check for `false` or `true`. Could do `not Persist` but `Persist =:= false` might read better